tests/conftest.py

```diff
@@ -20,6 +20,8 @@
 )
 from fmn.database.model import Destination, Filter, GenerationRule, Rule, TrackingRule, User
 
+pytest_plugins = "fmn.testing"
+
```

Comment:
This applies the above `register_message_class(autouse=True, ...)` fixture in all tests, not just those beneath `tests/consumer`, is that intentional?

Reply:
It's for the consumer tests and the rules tests now. I guess I could add it to `conftests.py` modules inside each, but the `register_message_class` fixture is pretty harmless. We may want to use this message class in the API too at some point when implementing the filtering preview feature anyway.